Podfile

```diff
@@ -13,4 +13,5 @@ pod 'CVCalendar', '~> 1.2.8'
 pod 'SimpleCircleProgressView', '~> 1.0.3'
 pod 'HorizontalProgressView', '~> 0.9.5'
 pod 'PickerSwift', '~> 0.9.3'
-pod 'DoneToolbarSwift', '~> 1.0.0'
\ No newline at end of file
+pod 'DoneToolbarSwift', '~> 1.0.0'
+pod 'GoogleMaps'
```

Comment:
To avoid API break, lock the version:

pod 'GoogleMaps' ~> '1.12.3' (the latest according to the terminal)

If 1 year from now we do pod install the code might not compile if we don't lock the version :)